pkgs/development/beam-modules/erlfmt/default.nix

```diff
@@ -0,0 +1,20 @@
+{ fetchFromGitHub, rebar3Relx, lib }:
+
+rebar3Relx rec {
+  name = "erlfmt";
```

Comment:
```suggestion
  pname = "erlfmt";
```

Reply:
rebar3Relx doesn't accept `pname`, probably something we should fix in another PR?

Reply:
Yeah, didn't remember that. @happysalada could you take a look at this? Everything should accept pname to make it uniform.